### DISCUSSION TITLE
Trying to run inference, but there is an issue with the last frame of one of my videos

### DISCUSSION THREAD

**Discussion by ainsleybarrow** [talmolab/sleap#1687] [Help!]
Hello, 

I previously created a model that I would like to apply to a new set of videos. I attempted to run inference, but there appears to be an issue with the last frame of one of my videos. I included a screenshot. Is there a way to fix this without getting rid of the last frame of this video? 

![sleap_issue](https://github.com/talmolab/sleap/assets/159499685/5c11ae7f-56c3-4373-a11d-f74dfd64dd20)

**Comment by Marielamend** [comment]
Hello @ainsleybarrow!

Here is an issue similar to yours! Give this a go and let us know how it goes!

> We have seen a few problems like this occur when the video is not [reliably seekable](http://blog.loopbio.com/video-io-1-introduction.html). Try [reencoding your video](https://sleap.ai/help.html#does-my-data-need-to-be-in-a-particular-format) and let us know how that works.
> 
> #### Update: If reencoding doesn't work, and results in `moov atom not found`
> As I was helping a user, we found that they had just copied the contents of a zipped file over without unzipping which resulted in first the `KeyError`, and then, when attempting to reencode, a `moov atom not found` error. After retrying but with **unzipping first**, we were able to get everything running smoothly.
> 
> Thanks, Liezl
> 
> Related: #767

Best,
Mariela
